core/src/main/java/com/venia/core/models/commerce/services/cacheinvalidation/CustomInvalidateTypeForDispatcherStrategy.java

```diff
@@ -24,12 +24,14 @@
  */
 @Component(
         service = DispatcherCacheInvalidationStrategy.class)
-public class CustomAttributeForClearCacheForGraphqlAndDispatcher implements DispatcherCacheInvalidationStrategy {
+public class CustomInvalidateTypeForDispatcherStrategy implements DispatcherCacheInvalidationStrategy {
```

Comment:
I would suggest: `CustomDispatcherInvalidation`. Please provide example request Json in the Javadoc how to call this invalidation.